### PR TITLE
Simplify Firefly dockerfile

### PR DESCRIPTION
**Pull request by loitly**
- remove unused variables
- predefine Firefly runtime directories, making it easier for user to mount into
- move statistics directory into logs/statistics
- VISUALIZE_FITS_SEARCH_PATH now defaults to blank.  Setting it will append to the '/external' set by default.

### CODE REVIEW

File: docker/base/launchTomcat.sh
Comment (robyww) [inline docker/base/launchTomcat.sh:54]: maybe we want to break these into two parts as well, common and advanced.
`logs`, `shared-workarea`, `/external` as common and the others as advanced.

File: docker/base/launchTomcat.sh
Comment (robyww) [inline docker/base/launchTomcat.sh:87]: good, I think this idea

File: docker/base/launchTomcat.sh
Comment (robyww) [inline docker/base/launchTomcat.sh:100]: since you made this change in statistics, does this mean that we will not attempt to make a statistics dir if the use just maps logs?

File: docker/base/start-examples.txt
Comment (robyww) [inline docker/base/start-examples.txt:46]: should we mentions starting a redis server and pointing to it here. I know we did not in the pass, but it is clearer now what to do.

**Review by robyww — APPROVED** [review]
This all looks good. I want to talk to you before you merge it though.

File: docker/base/launchTomcat.sh
Comment (loitly) [reply docker/base/launchTomcat.sh:100]: If they mount /firefly/logs, then we would create a statistics directory under logs if one does not exists. Permission is not a problem since we expect the process owner to have write access to this directory.
If they want only the stats logs to persistent outside the container, then they can mount /firefly/statistics instead.

File: docker/base/start-examples.txt
Comment (loitly) [reply docker/base/start-examples.txt:46]: > Should we mention starting a redis server and pointing to it here? I know we did not in the pass, but it is clearer now what to do.

I don't think so since redis is only needed when you have multiple instances of Firefly running.  These examples are for a single instance.
With multiple instances, you may need to introduce docker-compose or may be swarm.  Both, are out of our scope.

Comment (robyww) [reply docker/base/start-examples.txt:46]: ok

**Pull request merged**
